Text Formatter/js/textHandler.js

```diff
@@ -0,0 +1,57 @@
+class TextFormatter{
+
+    format(text, stringLength, stringCount){
+
+        let resultText = "";
+        for(let i = 0; i < text.length; i++){
+            if(i / stringLength < stringCount){
+                if(i % stringLength == 0){
+                    resultText += '\n';
+                }
+                resultText += text[i];
+            }
+        }
+        return resultText.substring(1);
+    }
+
+    formatByType(text, type){
+        switch(type){
+            case "Word":
+                return this.formatByWord(text);
+            case "Char":
+                return this.formatByChar(text);
+            case "Sentence":
+                return this.formatBySentence(text);
+            default:
+                return text;
+        }
+    }
+
+    formatByWord(text){
```

Comment:
string length and string count is not used in formating by word, char or sentence. the implementation is not correct.